tools/merge_schedule

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+
+# Tool to merge YAML_SCHEDULE_DEFAULT  with YAML_SCHEDULE_FLOWS
+# and YAML_SCH§EDULE.
+
+# So far all pointers should be provided as environment
+# variables
+
+use strict;
+use warnings;
+
+use File::Basename;
+use testapi qw(get_var set_var diag);
+use Env;
+
+use YAML::PP;
+use YAML::PP::Schema::Include;
+use YAML::PP::Common 'PRESERVE_ORDER';
+use scheduler;
+
+my $test_suite_data;
+my $root_project_dir = dirname(__FILE__) . '/../';
+my $include = YAML::PP::Schema::Include->new(paths => ($root_project_dir));
+my $ypp = YAML::PP->new(
+    schema => ['Core', $include, 'Merge'],
+    preserve => PRESERVE_ORDER
+);
+$include->yp($ypp);
+
+set_var('YAML_SCHEDULE', $ENV{YAML_SCHEDULE});
```

Comment:
please, do not use `set_var` in the script, use parameters, we should try to use minimum number of libraries for this scripts. Same apply for `scheduler`, you just need to copy/paste the specific code from scheduler not to use the whole thing, this is not for an unit test, it is more for running a script from your machine and see what is going on. No need for an export as it is not going to be useful, always will be created dynamically.

Reply:
If I copy/paste, then with the next change in the schedule it will diverge from what is in the script if nobody remembers to update the script as well. That's something I'm trying to avoid.

Reply:
discussed offline that we cannot use the project for this scripts, otherwise we should not include them in the repo, they are supposed to be independent utilities.